packages/flutter/lib/src/cupertino/segmented_control.dart

```diff
@@ -179,6 +180,11 @@ class CupertinoSegmentedControl<T> extends StatefulWidget {
   /// Defaults to the selectedColor at 20% opacity if null.
   final Color pressedColor;
 
+  /// The [children], if any, are placed inside this padding.
```

Comment:
Please tweak the comment here to make it very specific that this is about the padding around the whole CupertinoSegmentedControl and not around each individual segment.